site/en/guides/border-router/esp-br.md

```diff
@@ -0,0 +1,125 @@
+# ESP Thread Border Router
```

Comment:
The TOC for the border-router directory is being added in #42 - please add an entry to that file for this new guide.